students/BrandonHenson/Lesson4/trigrams.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+# Brandon Henson
+# 4/16/18
+# Lesson04
+# Trigram
+
+import random
+tridict = {}
+list1 = []
+
+
+def readdata():
+    with open('./s.txt', 'r') as fileobj:
+        read_data = fileobj.read()
+        words = read_data.replace('\n', ' ').replace('--', ' ')\
+            .replace('!', ' ').replace("'", ' ')\
+            .replace(',', ' ').replace('.', ' ').replace('  ', ' ').split(' ')
+        for word in words:
+            if not word == '':
+                list1.append(word)
```

Comment:
with is only really needed for file context work. you can de-dent lines 15 to 20